dist/src/main/config/broker.standalone.yaml.template

```
@@ -332,6 +333,19 @@
           # This setting can also be overridden using the environment variable ZEEBE_BROKER_DATA_BACKUP_S3_BASEPATH
           # basePath:
 
+        # Configure the following if store is set to GCS
+        # gcs:
+          # Name of the bucket where the backup will be stored.
+          # The bucket must already exist.
+          # The bucket must not be shared with other Zeebe clusters unless basePath is also set.
+          # This setting can also be overridden using the environment variable ZEEBE_BROKER_DATA_BACKUP_GCS_BUCKETNAME
+          # bucketName:
+
+          # When set, all blobs in the bucket will use this prefix. Must be non-empty and not start or end with '/'.
```

```suggestion
          # When set, all blobs in the bucket will use this prefix. Must be non-empty and not start or end with '/'.
```

For better UX we could also just remove the trailing `/` right?

---

:+1: I implemented and tested this in the last two commits